### ISSUE TITLE
Submodule library for cloudresourcemanager does not exist

### ISSUE THREAD

**Issue by Utsav2** [googleapis/google-api-nodejs-client#2586]
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

1) Is this a client library issue or a product issue?
This is the client library for . We will only be able to assist with issues that pertain to the behaviors of this library. If the issue you're experiencing is due to the behavior of the product itself, please visit the [ Support page]() to reach the most relevant engineers.

Client Library

2) Did someone already solve this?
  - Search the issues already opened: https://github.com/googleapis/google-api-nodejs-client/issues
  - Search the issues on our "catch-all" repository: https://github.com/googleapis/google-cloud-node
  - Search or ask on StackOverflow (engineers monitor these tags): http://stackoverflow.com/questions/tagged/google-cloud-platform+node.js

3) Do you have a support contract?
Please create an issue in the [support console](https://cloud.google.com/support/) to ensure a timely response.

No

If the support paths suggested above still do not result in a resolution, please provide the following details.

#### Environment details

  - OS:
  - Node.js version:
  - npm version:
  - `googleapis` version:

#### Steps to reproduce

  1. `npm install @googleapis/cloudresourcemanager` from https://github.com/googleapis/google-api-nodejs-client/tree/master/src/apis/cloudresourcemanager
  2. Not Found - GET https://registry.npmjs.org/@googleapis%2fcloudresourcemanager - Not found

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by JustinBeckwith** [comment]
Greetings!  We very specifically don't publish Cloud focused libraries from this repository into split package.  You can find a list of the cloud focused products here:
https://github.com/googleapis/google-cloud-node/

For Cloud Resource Manager - you likely want `@google-cloud/resource`:
https://github.com/googleapis/nodejs-resource

Curiosity - is there a specific reason you wanted to use `googleapis` to do this, or is it just the place you landed first?  We're trying to make this all easier to figure out.

**Comment by Utsav2** [comment]
I landed here since we already use `cloudresourcemanager` in production. 

Our use case is that we already use `googleapis`, but we'd like to use the smaller module so that `tsc` doesn't have to re-compile all of `googleapis` on clean builds. I looked at `@google-cloud/resource`, but it doesn't seem like there's a clear path of migration from `cloudresourcemanager`. Additionally, the `resource` library is considered "alpha", so we do not want to use that in production. We don't need it to be `googleapis` specifically.
